File: src/test/java/com/larryhsiao/clotho/color/GreenIntTest.java
package com.larryhsiao.clotho.color;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

/**
 * Test for {@link GreenInt}
 */
public class GreenIntTest {
    /**
     * Normal case
     */
    @Test
    void normalCase() {
        assertEquals(
            160,
            new GreenInt(0xFFA0FF).value()
        );
    }
}

File: src/test/java/com/larryhsiao/clotho/color/GreenIntTest.kt
package com.larryhsiao.clotho.color

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

/**
 * Test for [GreenInt]
 */
class GreenIntTest {
    @Test
    fun simple() {
        assertEquals(
            160,
            GreenInt(0xFFA0FF).value()
        )
    }
}